bitcoinplusplus/layer-2/lightning-on-taproot.md

```diff
@@ -253,10 +253,10 @@ So I think it's a major issue for the full kind of research and investigation on
 But That being said, another aspect where we can somewhat impropriate the STTLCs, because of the issue that I mentioned earlier, where payments in hops could not be correlated.
```

Comment:
We expect reviewers to catch this kind of AI's transcription errors. You are the one watching the videos so it's easier to make the connection that at this moment he is talking about "PTLCs" as shown by the speakers slides.
```suggestion
That being said, another aspect where we can somewhat improve privacy is PTLCs, because of the issue that I mentioned earlier, where payments in hops could not be correlated.
```